test/unit/deprecated.js

```diff
@@ -406,7 +406,7 @@ supportjQuery.each(
 	function( subclass, source ) {
 		var fn;
 		try {
-			fn = Function( "return " + source )();
+			fn = Function( "return " + source );
```

Comment:
This change is incorrect, `fn` is supposed to be a generator or async function in browsers that support those constructs. We just use `Function` to avoid syntax errors in the test suite itself for browsers that don't support them.

Reply:
I ran the testsuite locally and this caused also a failing unit test on your side.

Reply:
The "deprecated" module is passing in all browsers: http://swarm.jquery.org/job/6744